eiffel-vocabulary/EiffelTestCaseFinishedEvent.md

```diff
@@ -60,12 +60,12 @@ __Description:__ A list of metrics collected during the test case execution. Not
 
 ##### data.outcome.metrics.name
 __Type:__ String  
-__Required:__ No  
+__Required:__ Yes  
```

Comment:
Yes, but the problem is we are now changing the definition of the event, so we need to step the version. This is because the .md file is the canonical source of truth: either we think 1.0.0 is correct and the schema is wrong, or it needs to be changed. Clearly, your change is the right one (data.outcome.metrics is optional, but if used its properties shouldn't be).

In other words, what is needed is stepping the version, e.g.: 
`| 1.0.1     | Current version                                        | data.outcome.metrics.value and data.outcome.metrics.name made mandatory |`

Then a 1.0.1 version of the schema needs to be added, and the current 1.0.0 version "fixed" (i.e. the properties made optional).